scripts/plot_multiple_gcamps_2min.m

```diff
@@ -0,0 +1,43 @@
+clear; close all
+dir_of_files = uigetdir;
+D = dir(dir_of_files);
+s = what(dir_of_files);
+files = s.mat;
+GC = [];
+c = 1;
+filename = 'heatmap'
+for i = 1:numel(files)
+    load([dir_of_files '/' char(files(i))]);
+    GC(c).blc = blc(1:240);
+    GC(c).delta_F_mean = delta_F_mean(1:240);
+    GC(c).resampletemp = resampletemp(1:240);
+    c = c + 1;
+end
+
+bc = zeros(length(GC),240);
```

Comment:
Seems a little excessive to make a new script to change one variable. This is what arguments in functions are for.

Reply:
There are at least 10 of these.